src/controllers/v1/cloud-services.js

```diff
@@ -16,9 +16,10 @@ module.exports = class CloudServices {
 	 */
 	async getSignedUrl(req) {
 		try {
-			const signedUrlResponse = await filesService.getSignedUrl(
+			const signedUrlResponse = await filesService.getSignedUrlByOrgId(
```

Comment:
Why was this changed from getSignedUrl to getSignedUrlByOrgId()?

Reply:
@nevil-mathew Intially we were upload genric file for all the organizations , but the flow is changed to each organization can have it own custom csv.